extensions/commands/cci/cmd_list_v2_ready.py

```diff
@@ -13,6 +13,11 @@ def output_json(results):
     print(json.dumps(results, indent=2))
 
 
+def output_file_json(file_path, results):
```

Comment:
Command formatters shouldn't write to disk. Instead they should to print to stdout and file redirect to be used to create a file.